.github/workflows/async-stripe.yml

```diff
@@ -45,7 +45,7 @@ jobs:
         uses: actions-rs/cargo@v1
         with:
           command: make
-          args: openapi-install-no-fetch
+          args: openapi-install
```

Comment:
This is just a quickfix to address https://github.com/arlyon/async-stripe/commit/12bd6bae7ffa18dd701bc44a24a6a23b30e8be25#r88345568 so that the verify codegen CI step can (hopefully) work on this PR since it's an important check for a PR like this to verify nothing in what we generate is modified :)

Reply:
Yeah I decided to revert that so that change is effectively 'transparent' :)